code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -811,4 +811,6 @@
 	var/no_protection = FALSE
 	if(dna && dna.species)
 		no_protection = dna.species.handle_fire(src, seconds_per_tick, no_protection)
```

Comment:
Fairly certain `handle_fire` is officially unused with this pr so you can delete it entirely, it's trash anyays